docs: explain how to update the content of the marketplace (#36)
bonitasoft/bonita-marketplace: README.md

# bonita-marketplace

This repository lists the extensions available on the Bonita Marketplace:
- actor-filters
- applications
- connectors
- database-drivers

It specifies which extension versions are compatible with each Bonita version.

## Adding a New Extension or Version

To add a new extension or update the version of an existing one:

1. Update or add files in the related directory in `src/main/resources/artifacts`.
2. Commit and push your changes (prefer the usage of Pull Request).

## Releasing a New Version of Bonita Marketplace

To create a new release:

1. Run the [release GitHub action]([.github/workflows/release.yml](https://github.com/bonitasoft/bonita-marketplace/actions/workflows/release.yml)).
2. Select the release version by reviewing the [existing releases](https://github.com/bonitasoft/bonita-marketplace/releases) and incrementing as needed.

This will generate a new release with assets (zip file) containing the Bonita Marketplace content.

## Updating the Marketplace in Bonita Studio

> Bonita Studio ships with an offline copy of the marketplace, allowing users to install extensions without an internet connection. This copy is refreshed with each Bonita Studio release.

Reviewer: ```suggestion
Bonita Studio ships with an offline copy of the marketplace, allowing users to install extensions without an internet connection. This copy downloaded for each Bonita Studio.
```
